src/sentry/notifications/platform/discord/provider.py

```diff
@@ -68,8 +70,7 @@ def render[DataT: NotificationData](
 
         if len(rendered_template.actions) > 0:
             buttons = [
-                DiscordButton(
-                    custom_id=action.label.lower().replace(" ", "_"),
+                DiscordLinkButton(
```

Comment:
Maybe this is not a problem now for the default renderer. But currently we use 0 url/link buttons for Discord. All of our buttons are 'Interactive' buttons (i.e the flavor with a `custom_id` and no url), that then send a webhook to us and we do some action based off the custom_id. 

So this'll definitely be an issue when we try to migrate the issue notifications to the platform for discord/msteams, since our rendered template doesn't really account for this custom_id webhook flow. I think it'd be worth to add to the rendered_template actions a custom_id field since it looks like msteams & discord use some flavor of this mechanism for assignment.

idk where else i was going with this, other than I'm gonna add custom_id as a field to the rendered template lol